### PR TITLE
Add locale options to views

### PR DESCRIPTION
**Pull request by GuiHeurich**
Add locale options to views.

### CODE REVIEW

**Comment by GuiHeurich** [comment]
@matschaffer

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -3,9 +3,13 @@
...
     if I18n.locale == :'en-US'
-      link_to 'JA', locale: 'ja'
+      [
+        link_to('CZ', locale: 'cs-CZ'),
+        link_to('JA', locale: 'ja'),
+        link_to('PT', locale: 'pt')
+      ]
     else
-      link_to 'EN', locale: 'en-US'
+      [link_to('EN', locale: 'en-US')]
```
Comment (eitoball) [inline app/helpers/application_helper.rb:12]: Why can you choose `CZ`, `JA`, or `PT` when current locale is `en-US`, but only `EN` when locale is `ja`, `cs-CZ`, or `pt`?

Comment (GuiHeurich) [reply app/helpers/application_helper.rb:12]: But having all options all the time might be good too - and it would remove the need for an if statement.

Comment (matschaffer) [reply app/helpers/application_helper.rb:12]: ++ for allowing free exchange between any language :)

Comment (GuiHeurich) [reply app/helpers/application_helper.rb]: Done.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -2,11 +2,12 @@
...
-      link_to 'JA', locale: 'ja'
-    else
-      link_to 'EN', locale: 'en-US'
-    end
+      [
+        link_to('CZ', locale: 'cs-CZ'),
+        link_to('JA', locale: 'ja'),
+        link_to('PT', locale: 'pt'),
+        link_to('EN', locale: 'en-US')
+      ]
```
Comment (matschaffer) [inline app/helpers/application_helper.rb:10]:
```suggestion
    [
      link_to('CZ', locale: 'cs-CZ'),
      link_to('JA', locale: 'ja'),
      link_to('PT', locale: 'pt'),
      link_to('EN', locale: 'en-US')
    ]
```

**Review by eitoball — APPROVED** [review]

**Comment by matschaffer** [comment]
Thanks @GuiHeurich! 🧡

**Comment by GuiHeurich** [comment]
My pleasure, let me know on what I can work on next. :)

**Comment by BRSMaulwurf** [comment]
i´ve added the language "German" on locale a while ago. feel free to add it to the api 🧡

**Pull request merged**